src/inspector_agent.h

```diff
@@ -39,6 +39,7 @@ class InspectorSessionDelegate {
  public:
   virtual ~InspectorSessionDelegate() = default;
   virtual bool WaitForFrontendMessage() = 0;
+  // XXX SendMessageToFrontend() ?
```

Comment:
Got it. I'll rename, only takes a minute, and I promise I will feel no sadness if you delete any of the renamed functions.